esiclient/tests/unit/v1/test_node_console.py

```diff
@@ -0,0 +1,70 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+from osc_lib import exceptions
+
+from esiclient.tests.unit import base
+from esiclient.v1 import node_console
+
+
+class TestNodeConsoleConnect(base.TestCommand):
+
+    def setUp(self):
+        super(TestNodeConsoleConnect, self).setUp()
+        self.cmd = node_console.NodeConsoleConnect(self.app, None)
+
+        self.node1 = {
+            "uuid": "11111111-2222-3333-4444-aaaaaaaaaaaa",
+            "name": "node1",
```

Comment:
This represents the return value of `openstack baremetal node console show`, right? If so, I don't think it returns a uuid or name; just the console values below.